packages/mattermost-redux/src/reducers/entities/users.ts

```diff
@@ -458,25 +454,31 @@ function profilesNotInGroup(state: RelationOneToMany<Group, UserProfile> = {}, a
     }
 }
 
+function saveToState<T>(state: Record<string, T>, key: string, value: T): Record<string, T> {
```

Comment:
can we think of a better name, because its not actually saving but merging 0/5 🤔

Reply:
Maybe `addToState`? I just put this helper here, but it seems like it'd be really helpful throughout all the redux state. I'm pretty sure we could use these helpers in Joram's PR too, but I don't want to change too much else right now.